app/models/manageiq/providers/nuage/inventory/collector/target_collection.rb

```diff
@@ -37,6 +37,10 @@ def network_routers
     @network_routers_map.values.compact
   end
 
+  def floating_ips
+    [] # TODO(miha-plesko): implement targeted refresh for floating ips
```

Comment:
Please create issues for this so that we don't forget on them. I see another TODO above.